src/components/CookieClicker.vue

```diff
@@ -0,0 +1,411 @@
+        Grandma: {{ availableUpgrades[1].owned }} |
+        Farm: {{ availableUpgrades[2].owned }}
+      </div>
+
+      <img 
+        v-if="availableUpgrades[0].owned > 0"
+        :src="cursorImage"
+        class="upgrade-visual"
+        :style="getUpgradeStyle(0)"
+        @error="handleImageError('cursor')"
+      >
+      <img 
+        v-if="availableUpgrades[1].owned > 0"
+        :src="grandmaImage"
+        class="upgrade-visual"
+        :style="getUpgradeStyle(1)"
+        @error="handleImageError('grandma')"
+      >
+      <img 
+        v-if="availableUpgrades[2].owned > 0"
+        :src="farmImage"
+        class="upgrade-visual"
+        :style="getUpgradeStyle(2)"
+        @error="handleImageError('farm')"
+      >
+    </div>
+
+      <div class="upgrades-section">
+        <h2>Upgrades</h2>
+        <div class="upgrades-list">
+          <div 
+            v-for="(upgrade, index) in availableUpgrades" 
+            :key="'upgrade-' + index"
+            class="upgrade-card"
+            :class="{ 
+              'can-afford': cookies >= upgrade.cost,
+              'purchased': upgrade.owned > 0
+            }"
+            @click="purchaseUpgrade(index)"
+          >
+            <div class="upgrade-header">
+              <h3>{{ upgrade.name }}</h3>
+              <span class="owned-count" v-if="upgrade.owned > 0">×{{ upgrade.owned }}</span>
+            </div>
+            <div class="upgrade-stats">
+              <p class="cost">Cost: {{ formatNumber(upgrade.cost) }} cookies</p>
+              <p class="value">+{{ upgrade.value }} CPS</p>
+            </div>
+            <div class="upgrade-description">
+              <p>{{ upgrade.description }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+  
+
+      <transition name="slide-fade">
+        <div v-if="showAchievementNotification" class="achievement-notification">
+          <span class="badge">✓</span>
+          Achievement Unlocked: {{ achievementNotificationText }}
+        </div>
+      </transition>
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    name: 'CookieClicker',
+    data() {
+      return {
+        cookies: 0,
+        totalClicks: 0,
+        cookiesPerSecond: 0,
+        availableUpgrades: [
+          {
+            name: "Cursor",
+            description: "Automatic cookie clicking",
+            cost: 25,
+            value: 0.1,
+            owned: 0
+          },
+          {
+            name: "Grandma",
+            description: "A nice grandma to bake more cookies",
+            cost: 100,
+            value: 0.5,
+            owned: 0
+          },
+          {
+            name: "Farm",
+            description: "Grows cookie plants",
+            cost: 1100,
+            value: 4,
+            owned: 0
+          }
+        ],
+        showAchievementNotification: false,
+        achievementNotificationText: "",
+        cursorImage: require('../assets/upgrades/cursor.png'),
```

Comment:
use import over require and also actually then use this variables in src of image?